lib/engines/content_block_manager/test/unit/app/services/preview_content_test.rb

```diff
@@ -0,0 +1,54 @@
+require "test_helper"
+
+class ContentBlockManager::PreviewContentTest < ActiveSupport::TestCase
+  extend Minitest::Spec::DSL
+
+  let(:described_class) { ContentBlockManager::PreviewContent }
+  let(:host_content_id) { SecureRandom.uuid }
+  let(:preview_content_id) { SecureRandom.uuid }
+  let(:host_title) { "Test" }
+  let(:host_base_path) { "/test" }
+  let(:uri_mock) { mock }
+  let(:fake_frontend_response) do
+    "<body><p>test</p><span class=\"content-embed content-embed__content_block_email_address\" data-content-block=\"\" data-document-type=\"content_block_email_address\" data-content-id=\"#{preview_content_id}\">example@example.com</span></body>"
+  end
+  let(:block_render) do
+    "<span class=\"content-embed content-embed__content_block_email_address\" data-content-block=\"\" data-document-type=\"content_block_email_address\" data-content-id=\"#{preview_content_id}\">new@new.com</span>"
+  end
+  let(:block_render_with_style) do
+    "<span class=\"content-embed content-embed__content_block_email_address\" data-content-block=\"\" data-document-type=\"content_block_email_address\" data-content-id=\"#{preview_content_id}\" style=\"background-color: yellow;\">new@new.com</span>"
```

Comment:
If you made the `BLOCK_STYLE` constant public, you could do this:

```suggestion
    "<span class=\"content-embed content-embed__content_block_email_address\" data-content-block=\"\" data-document-type=\"content_block_email_address\" data-content-id=\"#{preview_content_id}\" style=\"#{ContentBlockManager::PreviewContent::BLOCK_STYLE}\">new@new.com</span>"
```

Reply:
hmm, I'm in two minds about that, because I quite like it when tests are more literal, and would break when that value is changed - but can use if you think it's more readable

Reply:
Nah, that's fair enough 👍